receiver/filelogreceiver/benchmark_test.go

```diff
@@ -0,0 +1,86 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package filelogreceiver
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+	"go.opentelemetry.io/collector/component/componenttest"
+	"go.opentelemetry.io/collector/consumer/consumertest"
+	"go.opentelemetry.io/collector/receiver/receivertest"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/pkg/stanza/operator/input/file"
+	"github.com/open-telemetry/opentelemetry-collector-contrib/receiver/filelogreceiver/internal/metadata"
+	"github.com/open-telemetry/opentelemetry-collector-contrib/receiver/filelogreceiver/internal/testutil"
+)
+
+func BenchmarkReadSingleStaticFile(b *testing.B) {
+	testCases := []struct {
+		numLines int
+	}{
+		{
+			numLines: 0,
+		},
+		{
+			numLines: 1,
+		},
+		{
+			numLines: 10,
+		},
+		{
+			numLines: 20,
```

Comment:
Seems like orders of magnitude is enough granularity. Would you be ok adopting the same format as here?

Reply:
Done in https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/38171/commits/30e70105289c3ebd26871a569f4cb689663f16ac.